src/txdb.cpp

```diff
@@ -28,6 +28,10 @@ static const char DB_FLAG = 'F';
 static const char DB_REINDEX_FLAG = 'R';
 static const char DB_LAST_BLOCK = 'l';
 
+//! Used to cache the nChainTx value for the base block of a UTXO snapshot.
+//! This only contains a value for chainstates created based on a snapshot.
+static constexpr char DB_NCHAINTX = 'n';
```

Comment:
Can't this be hardcoded beside the blockheight and assumeutxo hash in the chain params? 

(If not, I'll need to think more about how the txdb value for nchaintx interacts with major version upgrades, which bump the assumeutxo)